pkg/render/config.go

```diff
@@ -42,12 +43,20 @@ type AWSConfig struct {
 	WithCreds             bool   `json:"withCreds"`
 }
 
+// OpenStackConfig contains specific config for OpenStack
+type OpenStackConfig struct {
+	ClusterName           string `json:"clusterName"`
```

Comment:
you won't need these values, `ClusterName` can be something arbitrary and clusterID can be generated on the spec it self, right?

Reply:
ok ok, I'll remove it :smile: